aws/logs_monitoring/tests/snapshots/cloudwatch_log_custom_tags.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I think you also need to add a test case to `test_harness/tests/test_snapshots.py`